manim/mobject/types/point_cloud_mobject.py

```diff
@@ -214,9 +214,9 @@ class PointCloudDot(Mobject1D):
     def __init__(
         self,
         center=ORIGIN,
-        radius=0.075,
+        radius=2.0,
         stroke_width=2,
-        density=DEFAULT_POINT_DENSITY_1D,
+        density=10,
```

Comment:
Is `DEFAULT_POINT_DENSITY_1D` used in any other place? If not, please delete it (or change its value to `10` and use it here.)

Reply:
`DEFAULT_POINT_DENSITY_1D: int = 250` is defined in constants.py (line [104](https://github.com/ManimCommunity/manim/blob/fc754fa7f60a7b49840628311f809840c98cdceb/manim/constants.py#L104)) and only used once. Changing the variable to `10` and referencing it at `point_cloud_mobject.py` is the way to go.

Reply:
If it is used only once, it doesn't really make sense to keep the constant around (especially because I'm not sure that changing the value would have an effect on newly created objects).

Reply:
I'm ultimately neutral about this, though my personal preference is to keep the constant even if it is used only once, for semantics and readability reasons.

Reply:
`DEFAULT_POINT_DENSITY_1D` is seemingly used several time across the library, so I don't think changing it is the right thing to do.

That being said, I made this change because 250 (the original value of DEFAULT_POINT_DENSITY_1D) is way too much for `PointCloudDot`, causing it to be displayed incorrectly.

Reply:
I'm pretty sure it is [only used **once**](https://github.com/search?l=&q=DEFAULT_POINT_DENSITY_1D+repo%3AManimCommunity%2Fmanim&type=code) (in the current master branch). Instead of getting rid of that constant and hard-coding it to 10 in `class PointCloudDot`, I think the best move would be keeping the constant and changing it to 10, since other contributors can use that default value in the future for the same purpose.

Reply:
Oh, you're right. I will change this then, thanks:

Reply:
Should be fixed now.

Reply:
> for semantics and readability reasons

I won't block this PR over it, but I still think these sort of default constants are problematic and don't combine well with the removal of CONFIG dicts.

In particular, I don't see a difference between hard-coding the value in the constant vs. setting it as the default keyword argument.

In my understanding, changing the value of `DEFAULT_POINT_DENSITY_1D` in user code will not update the default value of the function, which is why the constant is rather confusing to me. (I didn't check, so I might be wrong about it.)